avoid downloading TensorFlow during the build, add placeholder to inject location of pre-downloaded TensorFlow sources
author: Denis Kristak (INUITS)
diff -ruN jax-jaxlib-v0.1.70_orig/WORKSPACE jax-jaxlib-v0.1.70/WORKSPACE
--- jax-jaxlib-v0.1.70_orig/WORKSPACE	2021-08-12 13:12:54.025608000 +0200
+++ jax-jaxlib-v0.1.70/WORKSPACE	2021-08-12 13:32:51.581961000 +0200
@@ -5,21 +5,20 @@
 # b) get the sha256 hash of the commit by running:
 #    curl -L https://github.com/tensorflow/tensorflow/archive/<git hash>.tar.gz | sha256sum
 #    and update the sha256 with the result.
-http_archive(
+# http_archive(
+#     name = "org_tensorflow",
+#     sha256 = "0f13410284b9186e436350e9617b3bed2d65f1dc1a220fd37ad9ef43c2035663",
+#     strip_prefix = "tensorflow-4039feeb743bc42cd0a3d8146ce63fc05d23eb8d",
+#     urls = [
+#         "https://github.com/tensorflow/tensorflow/archive/4039feeb743bc42cd0a3d8146ce63fc05d23eb8d.tar.gz",
+#     ],
+# )
+
+local_repository(
     name = "org_tensorflow",
-    sha256 = "0f13410284b9186e436350e9617b3bed2d65f1dc1a220fd37ad9ef43c2035663",
-    strip_prefix = "tensorflow-4039feeb743bc42cd0a3d8146ce63fc05d23eb8d",
-    urls = [
-        "https://github.com/tensorflow/tensorflow/archive/4039feeb743bc42cd0a3d8146ce63fc05d23eb8d.tar.gz",
-    ],
+    path = "pathToSed",
 )
 
-# For development, one can use a local TF repository instead.
-# local_repository(
-#    name = "org_tensorflow",
-#    path = "tensorflow",
-# )
-
 load("//third_party/pocketfft:workspace.bzl", pocketfft = "repo")
 pocketfft()
 
